### DISCUSSION TITLE
Integrate budibase app to another app

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#8090] [Help] [answered]
I have created one budibase application and also I have created another application using angular I want to "Integrate" budibase application to the Angular application. 
Is there any way to do that? 
like we can do with the iframe tag to show another application by using that application's url  can we do with same that thing or is there any another way way to do that

**Comment by andz-bb** [comment, marked as answer]
hey @DevrayePatil, there is an open issue for this here: #5659 - if you feel this would be useful you can drop a comment in there to show your support for it 👍

**Reply by DevrayePatil** [reply]
The solution is not present there

**Comment by savannt** [comment]
I'd recommend attempting to use iframe's and self hosting BB on same domain so CORS is not an issue,
Budibase Public API offers an endpoint to get login cookies

**Reply by gewubuntu** [reply]
where do i find this login cookie endpoint?

**Reply by savannt** [reply]
https://github.com/Budibase/budibase/discussions/6098#discussioncomment-3404094
